tensorboard/plugins/graph/graphs_demo.py

```diff
@@ -129,5 +128,18 @@ def g(i):
             tf.summary.trace_export("prof_g", step=step)
 
 
+def main():
+    # Create three demo graphs.
+    with _silence_deprecation_warnings():
```

Comment:
Hmm, could we scope the silence a little more narrowly to just the lines that require it?  That way we can still notice if there are new deprecation warnings on other symbols - the goal here should just be silence warnings that are unavoidable and known (like the ones about the profiler that come from the `tf.summary.trace_*` lines specifically).